mkdocs.yml

```diff
@@ -88,7 +89,7 @@ nav:
   - Contact Us: contact.md
 
 plugins:
-  - search
+  - material/search
```

Comment:
@jtherrmann the build failure is what I was seeing locally with this change as well

Reply:
looks like the Material Docs don't specify the `material/` prefix:
https://squidfunk.github.io/mkdocs-material/setup/setting-up-site-search/?h=search#built-in-search-plugin